### PR TITLE
Editorial: account for SharedArrayBuffer change in Web IDL

### PR DESCRIPTION
**Pull request by annevk**
See https://github.com/whatwg/webidl/pull/1311 for context.

At the same time adjust extract-idl.py to hold Python correctly ("!=" instead of "is not").

### CODE REVIEW

File: specs/latest/1.0/webgl.idl [hunk 1]
```diff
@@ -535,10 +535,11 @@ interface mixin WebGLRenderingContextBase
     const GLenum UNPACK_COLORSPACE_CONVERSION_WEBGL = 0x9243;
     const GLenum BROWSER_DEFAULT_WEBGL          = 0x9244;
 
-    [Exposed=Window] readonly attribute (HTMLCanvasElement or OffscreenCanvas) canvas;
-    [Exposed=Worker] readonly attribute OffscreenCanvas canvas;
+    readonly attribute (HTMLCanvasElement or OffscreenCanvas) canvas;
     readonly attribute GLsizei drawingBufferWidth;
     readonly attribute GLsizei drawingBufferHeight;
+    attribute PredefinedColorSpace drawingBufferColorSpace = "srgb";
+    attribute PredefinedColorSpace unpackColorSpace = "srgb";
```
Comment (kdashg) [inline specs/latest/1.0/webgl.idl:542]: How did this change get here?

**Review by kdashg — CHANGES_REQUESTED** [review]

**Comment by kdashg** [comment]
Thanks for this though! :)

File: specs/latest/1.0/webgl.idl (hunk 1 above)
Comment (annevk) [reply specs/latest/1.0/webgl.idl:542]: That's the result of running make?

Comment (kdashg) [reply specs/latest/1.0/webgl.idl:542]: Oh ok, so it was in the spec just not propagated out to the build outputs like the idl. Got it, thanks!

**Review by kdashg — APPROVED** [review]

**Comment by kdashg** [comment]
Can you sign the CLA? @annevk

File: specs/latest/1.0/webgl.idl (hunk 1 above)
Comment (annevk) [reply specs/latest/1.0/webgl.idl:542]: Yeah, though it's still bogus. Filed #3540.

**Comment by kdashg** [comment]
@annevk FWIW IIRC the CLA is just "we use the MIT license, and you grant copyright to Khronos", in case you are worried about how Formal and Serious that sounds. (but of course read it yourself to confirm!)

If that's not palatable, I offer to just fix this issue myself instead. :)

**Comment by grorg** [comment]
@kdashg Hi Kelsey. Unfortunately @annevk doesn't (yet) have permission to sign the CLA. I could repost this PR from my account, because I have signed the CLA. Or you could post it yourself. Let me know!

Either way, we offer this contribution under the Khronos CLA.

**Comment by kenrussell** [comment]
Thank you for this contribution and explicitly offering it under Khronos' CLA. To bypass the automated checks it's been integrated in #3560 .

**Comment by grorg** [comment]
Hmmm... I thought I commented here already.

@kdashg Anne isn't able to sign the CLA at the moment. Feel free to close this and fix it yourself, or I can post a copy (I've signed the CLA).

**Comment by grorg** [comment]
Oh, there it was. Something weird with my browser!

**Pull request closed without merging**